src/gui/SideBarWidget.cpp

```diff
@@ -62,16 +62,16 @@ void SideBarWidget::paintEvent( QPaintEvent * )
 
 	QFont f = p.font();
 	f.setBold( true );
-	f.setUnderline( true );
+	f.setUnderline(false);
 	f.setPointSize( f.pointSize() + 2 );
 	p.setFont( f );
 
 	p.setPen( palette().highlightedText().color() );
 
-	const int tx = m_icon.width()+4;
+	const int tx = m_icon.width() + 8;
 
 	QFontMetrics metrics( f );
-	const int ty = metrics.ascent();
+	const int ty = (metrics.ascent() + m_icon.height()) / 2;
```

Comment:
Not mandatory because all variables are int, but for sanity reasons I think it's better to static cast to int
const int ty = static_cast<int>(metrics.ascent() + m_icon.height()) / 2;

Reply:
It really wont matter much if they are both ``int``'s since no conversion will take place. The expression would return an ``int``, so there's no need. If it had returned a ``float``, then maybe. But if we don't need to use a cast, [we probably shouldn't](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es48-avoid-casts).